lib/base-compiler.js

```diff
@@ -476,11 +476,49 @@ class BaseCompiler {
         const addOpts = [];
         /* if not defined, consider it true */
 
+        // Build dump options to append to the end of the -fdump command-line flag.
+        // GCC accepts these options as a list of '-' separated names that may
+        // appear in any order.
+        var flags = '';
+        if (gccDumpOptions.dumpFlags.address !== false) {
```

Comment:
Is there a reason for the curiously specific `!== false` here? Is it specifically trying to avoid the normal "truthiness" of js objects?

Reply:
Forgive me: I see the original code was phrased this way! I don't know why, but as you can see I've merged and will update the site tonight!

Thank you again so much!

Reply:
> Forgive me: I see the original code was phrased this way! I don't know why, but as you can see I've merged and will update the site tonight!
> 
> Thank you again so much!

Yep, I just followed the existing style.